src/dataObjects/dtos/singleRights/DelegationInputDto.tsx

```diff
@@ -7,10 +7,12 @@ export interface DelegationInputDto {
 export class ServiceDto {
   serviceTitle: string;
   serviceOwner: string;
+  serviceType: string;
 
-  constructor(serviceTitle: string, serviceOwner: string) {
+  constructor(serviceTitle: string, serviceOwner: string, serviceType: string) {
```

Comment:
I started wondering if we could only show one action for Apps with a BFF solution instead. I want to massage the data in BFF more specifically for frontend in future so frontend can be more about just displaying the data it receives from BFF and do as little logic as possible. It made sense to do logic in frontend when we didn't have BFF, but now that we have. we need to make a transistion of how we do stuff.

The thing we would need to do here is making the service class actually do business logic and make apps just display action Access.

I just wanted to start a discussion about it because we have a lot of logic in frontend that we can start moving out to BFF. We can have a little discussion of pros and cons about doing it for this feature.

The biggest pro of this is that we don't get spread out logic in x numbers of components and Pages in frontend that it's hard to develop further in future because of all the dependencies.

Reply:
I'm not a fan of this logic being spread out all over the place either, but this is a temporary solution and, unfortunately, I don't think we can easily make the BFF handle the logic in this case 😅

Because even if we're displaying only a single action, we still need to _remember_ the actual actions so that we can delegate them later. If the BFF is to do this, it needs user-individual memory, which it doesn't have.

Which is why I think it's better to keep this logic in frontend

Reply:
Ahh that's true, however I wrote a draft user story that we can add tasks to when we come across some #667 
Just resolve this comment when read.